addons/web/controllers/main.py

```diff
@@ -655,6 +654,7 @@ def post(self, path):
             base_url = request.httprequest.base_url
             query_string = request.httprequest.query_string
             client = Client(request.httprequest.app, BaseResponse)
+            client = Client(http.root, BaseResponse)
```

Comment:
I think the previous line (656) needs to be removed.

Reply:
oups, correct, thanks!